src/linalg.jl

```diff
@@ -10,20 +10,31 @@ LinearAlgebra.logdet(B::BlockDiagonal) = sum(logdet, blocks(B))
 LinearAlgebra.tr(B::BlockDiagonal) = sum(tr, blocks(B))
 
 # Real matrices can have Complex eigenvalues; `eigvals` is not type stable.
-function LinearAlgebra.eigvals(B::BlockDiagonal; kwargs...)
-    # Currently no convention for sorting eigenvalues.
-    # This may change in later a Julia version https://github.com/JuliaLang/julia/pull/21598
-    return mapreduce(b -> eigvals(b; kwargs...), vcat, blocks(B))
+if VERSION < v"1.2.0-"
```

Comment:
Best to use a specific version number. You can use the `contrib/commit-name.sh` to convert the merge commit SHA to a Julia version

Reply:
Done :)